build/Dockerfile

```diff
@@ -13,21 +11,18 @@ LABEL 	io.k8s.display-name="OpenShift PSAP Ci artifacts" \
 COPY build/root /
 
 # Install openshift-ansible RPMs and some debugging tools
-RUN yum install -y epel-release \
-    && yum update -y \
-    && yum install --setopt=tsflags=nodocs -y \
-      jq wget git make vim \
-      inotify-tools ansible \
-      python3 openssh-clients \
+RUN yum install -y \
+        glibc-langpack-en \
+        git make jq vim wget \
+        python3 python3-devel python3-pip python3-setuptools \
+ && pip3 install --no-cache-dir --upgrade setuptools pip \
```

Comment:
Can we move `&&` to the end of the previous line?

Reply:
done